app/js/filters/tweetMention.js

```diff
@@ -9,8 +9,8 @@ var filtersModule = require('./_index.js');
 function tweetMentionFilter() {
 	return function(input) {
 		input = input.replace('</a>', '');
-		var mentionReg = /\B\@([\S\-]+)/gim;
-		var aTag = '<a class="external-mention" rel="external" href="./search?q=%40$1">@$1</a>';
+		var mentionReg = /\B\@([_a-z0-9\-]+)/gim;
+		var aTag = '<a class="external-mention" rel="external" href="./search?q=from:$1">@$1</a>';
```

Comment:
Looks right, but why would you have to change the regex ? What advantage does `_a-z0-9\-` give over `\S\-`
Can you point out to a failure case with `\B\@([\S\-]+)/gim` so that we can accept the regex change, the search change looks valid and correct.